src/components/place_form/PlaceForm.jsx

```diff
@@ -0,0 +1,222 @@
+import { useForm } from 'react-hook-form';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import ImageUploader from '../utils/UploadFile';
+import fetchCities from '../../redux/reservation_form/actions/fetchCities';
+import { sendPlaces } from '../../redux/places/placesSlice';
+
+const PlaceForm = () => {
```

Comment:
- [ ] When I try to add a new place, I try to upload a new image by pressing this button:

![image](https://github.com/grauJavier/final-capstone-front-end/assets/57421823/159123dc-d611-4cf3-ab97-69d50d315d51)

However, after selecting any image, this happens:


![image](https://github.com/grauJavier/final-capstone-front-end/assets/57421823/729b5a3c-bb96-4815-a263-d49ce5dcdc1c)

The screen suddenly turns blank and I get a console error:


![image](https://github.com/grauJavier/final-capstone-front-end/assets/57421823/ccab9d54-2040-4ac0-9f0d-b4b6f9f41464)

Kindly, fix this bug so your app runs as intended.  If this is not a bug, this means your app requires special procedures/configurations to be run correctly.  If this is the case, kindly add those instructions to your readme.

That way, your users will be able to add new places. 👍

Reply:
Hello!
Thanks for your feedback 🙌🏼
This is solved 😄